WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailsActivity.kt

```diff
@@ -0,0 +1,49 @@
+package org.wordpress.android.ui.history
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import org.wordpress.android.R
+import org.wordpress.android.analytics.AnalyticsTracker
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.ui.history.HistoryListItem.Revision
+
+class HistoryDetailsActivity : AppCompatActivity() {
```

Comment:
```suggestion
class HistoryDetailActivity : AppCompatActivity() {
```

Can we rename this class `HistoryDetailActivity` for consistency?